lib/src/main/main_bloc.dart

```diff
@@ -113,6 +116,7 @@ class MainBloc extends Bloc<MainEvent, MainState> {
     } else if (event is AppLoaded) {
       bool configured = await _context.isConfigured();
       if (configured) {
+        await _loadCustomerConfig();
```

Comment:
Loading and using the config here means adding the chat every time the login is performed right? Currently this should be okay, so no request to change something, but I will create a follow up ticket to split + document our "setup / login / loading logic" as this area feels a bit "wrong".